### PR TITLE
Renaming image names of waf readme

### PR DESCRIPTION
**Pull request by himsharma01**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/amazon-aws/waf.md [hunk 1]
```diff
@@ -112,19 +112,19 @@ Now that you have set up collection for AWS WAF, install the Sumo Logic App for
 
 See an overview of threats detected and traffic passing through AWS WAF.
```
Comment (JainM6) [inline docs/integrations/amazon-aws/waf.md:114]:
```suggestion

```

File: docs/integrations/amazon-aws/waf.md [hunk 2]
```diff
@@ -112,19 +112,19 @@ Now that you have set up collection for AWS WAF, install the Sumo Logic App for
...
 
-<img src={useBaseUrl('img/integrations/amazon-aws/AWS-WAF-Overview.png')} alt="AWS WAF" />
+<img src={useBaseUrl('img/integrations/amazon-aws/aws-waf-overview.png')} alt="AWS WAF" />
```
Comment (JainM6) [inline docs/integrations/amazon-aws/waf.md:116]:
```suggestion
```

File: docs/integrations/amazon-aws/waf.md [hunk 3]
```diff
@@ -112,19 +112,19 @@ Now that you have set up collection for AWS WAF, install the Sumo Logic App for
...
 
 
 ### AWS WAF Threat Intelligence
 
 See details of threats allowed and blocked by AWS WAF.
```
Comment (JainM6) [inline docs/integrations/amazon-aws/waf.md:121]:
```suggestion
```

File: docs/integrations/amazon-aws/waf.md [hunk 4]
```diff
@@ -112,19 +112,19 @@ Now that you have set up collection for AWS WAF, install the Sumo Logic App for
...
 
-<img src={useBaseUrl('img/integrations/amazon-aws/AWS-WAF-Threat-Intelligence.png')} alt="AWS WAF" />
+<img src={useBaseUrl('img/integrations/amazon-aws/aws-waf-threat-intelligence.png')} alt="AWS WAF" />
 
 
 
 ### AWS WAF Traffic
 
 See details of allowed and blocked AWS WAF traffic by location, rules and outliers.
```
Comment (JainM6) [inline docs/integrations/amazon-aws/waf.md:129]:
```suggestion
```

File: docs/integrations/amazon-aws/waf.md (hunk 1 above)
Comment (JainM6) [reply docs/integrations/amazon-aws/waf.md:114]: Extra line spaces

**Review by JainM6 — DISMISSED** [review]
Extra line spacing needs to be fixed, rest all looks good.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**